src/pages/Registration.jsx

```diff
@@ -55,13 +56,19 @@ function Registration() {
     } else if (password !== confirmPassword) {
       alert("Passwords do not match");
     } else {
-      // alert("Success");
-      console.log(user);
+      alert("Success");
       createUserWithEmailAndPassword(auth, email, password)
-        .then((userCredential) => {
+        .then(async (userCredential) => {
           // eslint-disable-next-line no-shadow
           const { user } = userCredential;
           console.log(user);
+          await addDoc(collection(db, "users"), {
+            firstName,
+            lastName,
+            email,
+            city,
```

Comment:
city isn't required, whats the use anyway.. storing extra data

Reply:
To prevent false location claims, users should only attend events in their actual place of residence. Otherwise, it can cause issues for other members. If a user cannot attend, it is best for them not to join the event at all.